staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/patch_customresource_handler.go

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
I see this file is part of an UPSTREAM (non carry) commit. If we intend to upstream this, the file should have a better name. I know @sttts suggested doing it this way - we need to resolve how we want to handle this.

Reply:
Maybe in this case I simply rename it. @sttts what's your preferred way in this specific simple case ?

Reply:
Would use the patch file. Value of upstreaming is low, and chance is that ppl don't like leaking internals. Not a war worth to fight for now IMO.